docs/metaed-project/DomainEntity/LMSSection.metaed

```diff
@@ -40,3 +40,8 @@ Domain Entity LMSSection
     datetime DeletedAt
         documentation "The date and time at which a record was detected as no longer available from the source system, and thus should be treated as 'deleted'."
         is optional
+    string SectionIdentifier
+        documentation "A mapping from this LMSSection to an EdFi SectionIdentifier, part of the identity of an EdFi Section"
+        is optional
+        role name EdFi
+        max length 255
```

Comment:
Should this be the `EdFiSectionId` instead, pointing to `edfi.Section.Id`? So that implementations where the section identifiers don't match up perfectly can have their own algorithm for determining how to match up records from these two tables, and then preserve this unique identifier for linking them.